tests/server.https.test.ts

```diff
@@ -0,0 +1,114 @@
+// tests/server.https.test.ts
+// Tests for the serve with HTTPS.
+
+import { afterEach, describe, test, expect, vi } from 'vitest';
+import { extend as createFetch } from 'got';
+
+import { loadConfiguration } from '../source/utilities/config.js';
+import { startServer } from '../source/utilities/server.js';
+
+// The path to the fixtures for this test file.
+const fixture = 'tests/__fixtures__/server/';
```

Comment:
Then you could change this to:

```suggestion
const fixtures = 'tests/__fixtures__/server/ssl/';
```

And use that in the tests below.